go/database/mpt/io/log.go

```diff
@@ -30,9 +30,11 @@ func NewLog() *Log {
 
 // Print logs a message that includes the time elapsed since the start of the program.
 func (l *Log) Print(msg string) {
-	now := time.Now()
-	t := uint64(now.Sub(l.start).Seconds())
-	l.logger.Printf("[t=%4d:%02d] - %s\n", t/60, t%60, msg)
+	if l != nil {
```

Comment:
How can l be nil?

Reply:
Yeah, I think the interface is what we normally use to create a 'Nil' instance of something.